### ISSUE TITLE
bug: Report inference should be sigmoid, not softmax

### ISSUE THREAD

**Issue by jakehlee** [emit-sds/cover-class#57]
For tasks doing multi-class multi-label trained with `BCEWithLogitsLoss`, `torch.sigmoid()` should be applied to the model output instead of `torch.nn.functional.softmax()`. Softmax forces the model to only predict a single class.

Opening this as an issue instead of PR'ing so that this can be folded into any other changes you plan on making to the Report.

https://github.com/emit-sds/cover-class/blob/7f700795e8960ae843a3cd4b03bddec511932967/src/cover_class/reporting/report_config.py#L90-L96
